src/include/distributed/backend_data.h

```diff
@@ -73,6 +74,11 @@ extern void DecrementExternalClientBackendCounter(void);
 extern bool ExecuteRemoteQueryOrCommand(char *nodeName, uint32 nodePort,
 										char *queryString, StringInfo queryResultString,
 										bool reportResultError);
+extern void AdjustAndInitializeClusterClock(void);
+extern void AdjustLocalClockToGlobal(void);
+extern uint64 CurrentClusterClockValue(void);
+extern uint64 GetEpochTimeMs(void);
+extern uint64 GetClusterClock(void);
 
```

Comment:
do we really need to export all those functions ?

Reply:
I know what happened, the PG_RETURN_UINT64  code was part of the SQL proc, I split it into two. Fixed now.

Reply:
GetEpochTimeMs --> removed it

1```
 metadata_utility.c RegisterAndAdjustClockValue     2223 AdjustAndInitializeClusterClock();
1 metadata_cache.c UpdateClockCatalog       5019 uint64 clockValue = CurrentClusterClockValue();
1 metadata_cache.c UpdateClockCatalog       5019 uint64 clockValue = CurrentClusterClockValue();

```

Reply:
given that we don't export GetEpochTimeMs() anymore (since it's removed), this is fine now.